functions/update-google-events.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
This could be covered later on by the bot that updates dependencies, once I've managed to get it to create one PR per "product" (or at least top level folder) so it can be assigned to owners.

Reply:
Right - I'll typically want to be able to do this in a controlled way, often just after releasing. We can delete it if we want though :)